site/en/blog/private-prefetch-proxy/index.md

```diff
@@ -0,0 +1,137 @@
+---
+# Required
+layout: 'layouts/blog-post.njk'
+title: Private prefetch proxy in Chrome
+description: >
+  Speeding up Largest Contentful Paint with cross-site prefetching.
+subhead: >
+  Speeding up Largest Contentful Paint with cross-site prefetching
+date: 2022-05-11
+authors:
+  - katiehempenius
+  - kenjibaheux
+  - buettner
+tags:
+  - privacy
+  - security
+hero: 'image/kheDArv5csY6rvQUJDbWRscckLr1/MKR7llxWeUvVif0CDGPM.jpg'
+alt: >
+  A gate with a sign reading private.
+---
+
+Since Chrome 92, we've been running an early access program (EAP) for a feature called private prefetch proxy, which speeds up out-going navigations from Google Search by 30% at the median. This private prefetch proxy feature allows the prefetching of cross-origin content without exposing user information to the destination website until the user navigates. We expect that the feature will graduate from this EAP as early as Chrome 103, thereby no longer requiring websites to opt-in, and allowing other referrer websites to safely speed up cross-site navigations.
+
+Read on to learn about [how this feature works](#how), 
+[how it can help significantly improve your sites' Largest Contentful Paint](#owners), 
+or [how referrer websites can help their users](#referrers) achieve their goals by speeding up cross-site navigations.
+
+## How private prefetch proxy works {: #how }
+
+### Secure communication channel
+
+This feature uses a [CONNECT](https://tools.ietf.org/html/rfc7231#section-4.3.6) proxy to establish a secure communication channel between Chrome and the server hosting the content to be prefetched. This secure communication channel prevents the proxy from inspecting any data transfer. Notably, while the private prefetch proxy necessarily sees the host name in order to establish a secure communication channel, it does not see the full URLs, nor the resources themselves.
+
+<figure>
+{% Img src="image/kheDArv5csY6rvQUJDbWRscckLr1/UF6dpd6QvSzGXXlM9YWE.gif", alt="Animation showing flow of data through proxy.", width="600", height="365" %}
+  <figcaption>Prefetching websites via a CONNECT proxy prevents leaking user information.</figcaption>
+</figure>
+
+In addition, because the secure communication channel is encrypted end-to-end, intermediaries can neither observe the host names, nor the content of the prefetched sites. Finally, the proxy inherently prevents the destination server from seeing the user's IP address.
+
+### Preventing user identification
+
+Beyond the network aspects detailed earlier, we also need to prevent servers from identifying a user at prefetch time, via information previously stored on their device. To that end, Chrome currently restricts the usage of Private Prefetch Proxy to websites for which the user has no cookies or other local state. Here are the restrictions for prefetch requests made via the private prefetch proxy:
```

Comment:
```suggestion
Beyond the network aspects detailed earlier, we also need to prevent servers from identifying a user at prefetch time, via information previously stored on their device. To that end, Chrome currently restricts the usage of Private Prefetch Proxy to websites for which the user has no cookies or other local state. Here are the restrictions for prefetch requests made via Private Prefetch Proxy:
```

Reply:
I think normalizing this helps. I get why "the private prefetch proxy" isn't capitalized, but this minor change improves consistency IMO.